.solhintignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 contracts/UsingWitnet.sol
+contracts/patterns/Payable.sol
 test/TestBuffer.sol
 test/TestActiveBridgeSet.sol
```

Comment:
Any reason to have these 4 lines here? I tried removing them and the only warning is about the `WitnetRequestSolved` modifier in `UsingWitnet.sol`, which could be renamed to `witnetRequestSolved`.

Reply:
Not any more, really. Most of the lines were due to previous previous. Payable can be also removed after last modification attending one of cbartoli's comments